offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -0,0 +1,449 @@
+package life.qbic.portal.offermanager.offergeneration
+
+import life.qbic.business.offers.Converter
+import life.qbic.business.offers.Currency
+import life.qbic.datamodel.dtos.business.Offer
+import life.qbic.datamodel.dtos.business.ProductItem
+import life.qbic.datamodel.dtos.business.services.*
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+
+import java.text.DecimalFormat
+
+/**
+ * <h1>Describes the quotation details of an offer</h1>
+ *
+ * <p>The quotation details section consists of the product items of an offer. It gives an overview of the purchased items and breaks down the total costs.
+ * This class solely generates the HTML source code for this section.</p>
+ *
+ * <p>General idea: group the product items, generate the td elements.
+ *
+ * Furthermore: This class stores fixed ids (such as QuotationOverview) to quickly access e.g. total costs,....
```

Comment:
I don't think this is true anymore. This class accesses fixed ids in the template to add the table elements
```suggestion
```